app/assets/stylesheets/layout.scss

```diff
@@ -0,0 +1,63 @@
+#outer-container, #topnav-container, #search-navbar-container, #sul-footer-container {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line